install_config/aggregate_logging.adoc

```diff
@@ -1101,6 +1101,27 @@ You can see the user interface by visiting the site specified by the
 See the link:https://www.elastic.co/guide/en/kibana/4.5/discover.html[Kibana
 documentation] for more information on Kibana.
 
+[[kibana-virtualizations-dashboard]]
+*Kibana Visualizations and Dashboard*
+
+Kibana visualizations and dashboard for monitoring container and pod logs allows
+administrator users (`cluster-admin` or `cluster-reader`) to view logs by
+deployment, namespace, pod, and container.
+
+Kibana visualizations and dashboard exists inside the Elasticsearch and ES-OPS
+pod, and must be run inside those pods. Log in to Kibana (or Elasticsearch)
+before running this script, because it requires some indices and other objects
+set up by the OpenShift Elasticsearch plugin.
+
+This script will also add an index pattern for `project.*` and load the
+necessary index pattern file.
+
+To load dashboards and other Kibana UI objects, run:
```

Comment:
To load dashboards and other Kibana UI objects, you must first log into Kibana as the user you want to add the dashboards to, then log out.  This will create the necessary per-user configuration that the next step relies on.  Then, run:

Where `$espod` is the name of one (any of them, shouldn't matter) of your Elasticsearch pods.